doc/api/v0.yaml

```diff
@@ -227,12 +227,47 @@ components:
         fund_id:
           type: integer
           format: int32
+        challenge_id:
+          type: integer
+          format: int32
+        challenge_type:
+          $ref: "#/components/schemas/ChallengeType"
```

Comment:
Perhaps it's better to group these in an object that shares part of the structure with `Challenge`, also adding the challenge title to facilitate use in an UI view.

Reply:
Or would be better, but then the discriminator thing will not work.